enforcer/src/main/java/org/wso2/micro/gateway/enforcer/util/TLSUtils.java

```diff
@@ -136,4 +139,19 @@ private static void updateTruststoreWithMutlipleCertPem (KeyStore trustStore, St
             log.error("Error while adding certificates to the truststore.", e);
         }
     }
+
+    public static Certificate getCertificate() {
+        Certificate certificate = null;
+        try {
+            CertificateFactory fact = CertificateFactory.getInstance(X509);
+            FileInputStream is = new FileInputStream(ConfigHolder.getInstance().getConfig().
+                    getPublicCertificatePath());
+            X509Certificate cert = (X509Certificate) fact.generateCertificate(is);
+            certificate = (Certificate) cert;
+            return certificate;
+        } catch (CertificateException | FileNotFoundException e) {
+            log.debug("Error in loading certificate");
```

Comment:
Here also if this is a util. This has to throw the error. If there is a specific reason to not to throw the error shall we add a comment with the reason